test/types/string/dmk/substring/substringBadRange2-codepoint.chpl

```diff
@@ -0,0 +1,3 @@
+var str = "åbçdèfghíjklm";
+
+writeln("str[13:codePointIndex..(-1):codePointIndex] = ", str[13:codePointIndex..(-1):codePointIndex]);
```

Comment:
It seems a little surprising that slicing by an empty range can result in an out of bounds error, depending on which empty range you're slicing by.  Arrays don't have that behavior, though I see slicing strings by byte offset ranges do.